src/main/java/johnny/InvalidArgumentsException.java

```diff
@@ -0,0 +1,21 @@
+package johnny;
+
+public class InvalidArgumentsException extends Exception{
```

Comment:
```suggestion
public class InvalidArgumentsException extends Exception {
```
Missing whitespace between Exception and the curly brace.